### PR TITLE
Create central configuration for more focused repo syncs

### PR DESCRIPTION
**Pull request by DavidBoike**
Would need to be merged at the same time https://github.com/Particular/InternalAutomation/pull/496 is deployed.

### CODE REVIEW

**Comment by adamralph** [comment]
> Would need to be merged at the same time #82 is deployed.

@DavidBoike that's a self reference. What did you intend to reference?

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  # No GitHub release artifacts to scan
+  - .github/workflows/virus-scan.yml
+excludeOnReleaseBranch:
+  # GitHub only reads this info on the default branch
+  - .github/SUPPORT.md
+  - .github/ISSUE_TEMPLATE/config.yml
+  - .github/ISSUE_TEMPLATE/bug_report.yml
+  - .github/ISSUE_TEMPLATE/feature_request.yml
+  - .github/ISSUE_TEMPLATE/improvement_request.yml
+  - .github/workflows/stale.yml
```
Comment (adamralph) [inline _config.yml:21]: Do we not want to mark stale PR's on release branches? Or do we think it's so rare as to not warrant the resource to run the workflow (does that matter to us?)?

**Comment by DavidBoike** [comment]
Whoops, should be https://github.com/Particular/InternalAutomation/pull/496 - I fixed the issue description as well.

File: _config.yml (hunk 1 above)
Comment (DavidBoike) [reply _config.yml:21]: That workflow runs on a cron trigger, so the only version of the file that matters is the version on the default branch.

Comment (adamralph) [reply _config.yml:21]: Ah, so the job on the main branch does all PR's, regardless of target branch. Gotcha.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# RepoStandards
+
+TODO: Documentation for the RepoSync process
```
Comment (andreasohlund) [inline README.md:3]: Do we want to add something here?

**Review by andreasohlund — APPROVED** [review]

**Pull request merged**